packages/view-types/feature-list/src/FeatureList.js

```diff
@@ -66,6 +67,16 @@ export default function FeatureList(props) {
     setSearchTerm(event.target.value);
   };
 
+  let columns, columnLabels;
+
+  if (showTable) {
+    columns = ['name', 'key'];
+    columnLabels = ['Gene Symbol', 'Ensembl ID'];
```

Comment:
Can we use `${capitalize(featureType)} ID` (pass down from parent `FeatureListSubscriber`) to make it generalizable to non-gene features?

For 'Ensembl ID', can we use `'Alternate ID'` to generalize to other use cases?

In the future we could perhaps make this configurable or use the column names from the AnnData/CSV but I think this would be enough for now.

Reply:
Of course, thanks for the suggestion. I have implemented this in this commit: 28eb48df404d85d7d94a17d875550e92161c37db Let me know something more to it needs adjustment.